console-subscriber/src/aggregator/mod.rs

```diff
@@ -269,6 +270,9 @@ impl Aggregator {
             .drop_closed(&mut self.resource_stats, now, self.retention, has_watchers);
         self.async_ops
             .drop_closed(&mut self.async_op_stats, now, self.retention, has_watchers);
+        if !has_watchers && !self.poll_ops.is_empty() {
+            self.poll_ops.clear();
```

Comment:
nit, take it or leave it: I believe `clear` is a no-op if the collection is empty, so it should be more or less equivalent to just write
```suggestion
        if !has_watchers {
            self.poll_ops.clear();
```

Reply:
Nice! I originally wrote it like that but then was worried about the cost of clear. Good to know, will add thanks.